example.py

```diff
@@ -5,4 +5,4 @@ def subtract(a,b):
   return a - b
 
 def multithingy(a,b):
-  return a*a
+  return a*b
```

Comment:
yeah this makes more sense! thank you marco!

Reply:
you're very welcome!!!!!!!!!!!